tests/e2e/machine_pool_test.go

```diff
@@ -47,22 +44,23 @@ var _ = Describe("TF Test", func() {
 		})
 		Context("Author:amalykhi-High-OCP-64757 @OCP-64757 @amalykhi", func() {
 			It("Create a second machine pool", func() {
-				token := os.Getenv(con.TokenENVName)
```

Comment:
Maybe I missed something, but, after removing that line there is no assignment/declaration for `token`

Reply:
It was added to Test initialization step (that runs before the tests) by Shelly in previous PR
https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/tests/e2e/init_test.go#L18